styles/syntax/ng.less

```diff
@@ -0,0 +1,47 @@
+.syntax--ng {
+  &.syntax--interpolation {
+    color: @hue-5;
+
+    &.syntax--begin, &.syntax--end {
+      color: @hue-2;
+    }
+
+    .syntax--function {
+      color: @hue-5;
+
+      &.syntax--begin, &.syntax--end {
+        color: @hue-2;
+      }
+    }
+
+    .syntax--bool {
+      color: @hue-6;
+    }
+
+    .syntax--bracket {
+      color: @mono-1;
+    }
+  }
+
+  &.syntax--pipe, &.syntax--operator {
+    color: @mono-1;
+  }
+
+  &.syntax--tag {
+    color: @hue-1;
+  }
+
+  &.syntax--attribute-with-value {
+    .syntax--attribute-name {
+      color: @hue-6-2 !important;
```

Comment:
Is `!important` needed here? I tested it with this [example](https://github.com/drootz/language-html-angular/blob/f3a36caf20ef64125adc418f1da57414b35fafd0/example/example.html) and haven't seen a difference.

To increase specificity, `.syntax--attribute-name` could also be repeated, like 

```
.syntax--attribute-name.syntax--attribute-name {
```

Then it's still possible to use `!important` in user's styles.less file to override it.

Reply:
Yep, not sure why I had that there, works fine without.